test/performance/vectorization/vectorizeOnly/vectorizeOnlyWithIntents.bad

```diff
@@ -0,0 +1,6 @@
+vectorizeOnlyWithIntents.chpl:5: syntax error: near 'with'
```

Comment:
I guess I'm surprised because I would have hoped that `foreach` would just copy the `forall` syntax productions. Maybe after concluding the "unify the intents" discussion we could also unify the grammar rules for most loop forms? Or maybe someone who touches the grammar frequently could unify them now and replace the errors with something else besides "syntax error"? (me)

Reply:
I think the history here was that `for` loops haven't traditionally supported intents and `foreach` based its implementation more on `for` than `forall`?  But we are talking about adding intents for `for` loops, which I think(?) has generally been gaining favor (see https://github.com/chapel-lang/chapel/issues/17857).  And even if we didn't support them for `for` loops, I think it makes sense to for `foreach` given that they're parallel ( #18500 ).

If you were to put a nicer user error together, that would be nice.  That said, it's been in this state for awhile now and hasn't really bothered anybody that I'm ware of.